pkg/util/net/ip/ip.go

```diff
@@ -82,3 +85,28 @@ func isIPv6Disabled(filename string) bool {
 
 	return data[0] == '1'
 }
+
+func ChangeCIDRsToIPs(ifs *api.InterfaceStatus) error {
+	// if IP address was the passed value we should do nothing
+	if net.ParseIP(ifs.Ip) == nil {
+		ip, _, err := net.ParseCIDR(ifs.Ip)
+		if err != nil {
+			return fmt.Errorf("qemu-guest-agent reported invalid CIDR value '%s' for '%s': %v",
+				ifs.Ip, ifs.InterfaceName, err)
+		}
+		ifs.Ip = ip.String()
+	}
+
+	for i, v := range ifs.IPs {
+		if net.ParseIP(v) == nil {
+			ip, _, err := net.ParseCIDR(v)
+			if err != nil {
+				return fmt.Errorf("qemu-guest-agent reported invalid CIDR value '%s' for '%s': %v",
+					v, ifs.InterfaceName, err)
+			}
+			ifs.IPs[i] = ip.String()
+		}
```

Comment:
Can we wrap this at a helper function (now that we even add a fallback to net.ParseIP)